### PR TITLE
Run code formatter on lib/mix/lib/mix/utils.ex

### PR DESCRIPTION
**Pull request by maartenvanvliet**
Per #6643

### CODE REVIEW

File: lib/mix/lib/mix/utils.ex [hunk 1]
```diff
@@ -401,10 +432,14 @@ defmodule Mix.Utils do
...
       {:error, _} ->
-        {:ok, File.cp_r!(source, target, fn(orig, dest) ->
-          File.stat!(orig).mtime > File.stat!(dest).mtime
-        end)}
+        {
+          :ok,
+          File.cp_r!(source, target, fn orig, dest ->
+            File.stat!(orig).mtime > File.stat!(dest).mtime
+          end)
+        }
```
Comment (whatyouhide) [inline lib/mix/lib/mix/utils.ex]: Please extract to variables and write the tuple on a single line as described in #6643.

File: lib/mix/lib/mix/utils.ex [hunk 2]
```diff
@@ -452,27 +492,30 @@ defmodule Mix.Utils do
...
+        {
+          :checksum,
+          """
           Data does not match the given SHA-512 checksum.
 
           Expected: #{expected}
             Actual: #{actual}
-          """}
+          """
+        }
```
Comment (whatyouhide) [inline lib/mix/lib/mix/utils.ex]: Same here.

**Comment by maartenvanvliet** [comment]
Fixed

File: lib/mix/lib/mix/utils.ex [hunk 3]
```diff
@@ -161,15 +165,20 @@ defmodule Mix.Utils do
...
   @doc false
   def last_modified_and_size(path) do
-    now = :calendar.universal_time
+    now = :calendar.universal_time()
 
     case :elixir_utils.read_mtime_and_size(path) do
       {:ok, mtime, size} when mtime > now ->
-        Mix.shell.error("warning: mtime (modified time) for \"#{path}\" was set to the future, resetting to now")
+        Mix.shell().error(
+          "warning: mtime (modified time) for \"#{path}\" was set to the future, resetting to now"
```
Comment (josevalim) [inline lib/mix/lib/mix/utils.ex]: Please extract the message to a variable.

**Comment by josevalim** [comment]
Hi @maartenvanvliet, there are a couple more comments from @whatyouhide and myself pending. Thank you!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**